Implement skip 15 seconds, back 15 seconds

lineville opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

Adds two new commands to the `spotify-player` extension to "Skip 15 seconds" and "Back 15 seconds" which allow users to quickly skip and rewind through Spotify songs and episodes.

## Screencast

https://github.com/user-attachments/assets/79381423-902f-4653-8d58-23a9219f4420


<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

thomaslombart reviewed (commented):
Thanks for the PR! 🙌
Can't that be used for songs as well? Or is it just for episodes? I'm a bit confused with the naming.

lineville commented:
@thomaslombart any concerns or suggested changes to this PR? Just curious if the project operates on a continuous release process or if this will get bundled in with several other changes before cutting a release?

thomaslombart reviewed (commented):
Looks almost good to me. The only feedback I have is to let the user execute the `Back 15s` or `Skip 15s` for a song; it doesn't really hurt to let them do that if they want. I agree it doesn't make sense to show these controls in the menu bar extension for tracks, though. While you're at it, could you update the changelog to today's date? 🙂

lineville commented:
@thomaslombart Sounds good, just pushed up those suggested changes! I've removed the guard against executing the commands for songs, and updated the changelog and Readme.

thomaslombart reviewed (approved):
🚀
